docker-compose.yml

```
@@ -0,0 +1,58 @@
+version: '3'
+
+services:
+  # WEB
+  web:
+    container_name: web
+    image: jinlee1703/yulion-web
+    command: npm start
+    ports:
+      - "3000:3000"
+  # RDB
+  rdb:
+    container_name: rdb
+    image: mysql/mysql-server:8.0.30
+    environment:
+      MYSQL_DATABASE: yulion
+      MYSQL_ROOT_HOST: '%'
+      MYSQL_ROOT_PASSWORD: abc123
+      TZ: 'Asia/Seoul'
+    ports:
+      - "3306:3306"
+    volumes:
+      - ./mysql/conf.d:/etc/mysql/conf.d
+    command:
+      - "mysqld"
+      - "--character-set-server=utf8mb4"
+      - "--collation-server=utf8mb4_unicode_ci"
+  cache:
+    container_name: cache
+    image: redis:alpine
+    command: redis-server --requirepass abc123 --port 6379
+    hostname: redis_boot
+    labels:
+      - "name=redis"
+      - "mode=standalone"
+    ports:
+      - 6379:6379
+  # WAS
+  was:
+    container_name: was
+    image: jinlee1703/yulion-was
+    ports:
+      - "8080:8080"
+    environment:
```

이 부분은 아래처럼 해보시면 어떠신가요?
.env 파일을 docker-compose.yml과 같은 경로에 놔두시고 .env 파일에 아래 내용을 적으시면 도커에 환경 변수들을 주입할 수 있어요!
```suggestion
    environment:
      .env
```

---

엇 docker-compose가 올라가버렸네요. 브랜치가 꼬여버렸군요 ㅠㅠ 말씀해주신 내용 반영해서 수정해보겠습니다!

---

앗 그렇군요! 저는 docker compose 도 깃헙에 같이 올리면 실행시켜주는 건지 알았는데 잘못 알고 있었네요!
그러면 .env 파일은 서버 상의 docker-compose.yml이랑 같은 디렉토리에 나두면, 실행할때 env 파일을 같이 읽을 것 같아요